src/pybind/CMakeLists.txt

```diff
@@ -58,4 +58,4 @@ file(COPY ${NMODL_PROJECT_SOURCE_DIR}/nmodl/ext DESTINATION ${CMAKE_BINARY_DIR}/
 # =============================================================================
 # Install python binding components
 # =============================================================================
-install(DIRECTORY ${CMAKE_BINARY_DIR}/lib/ DESTINATION lib)
+install(DIRECTORY ${CMAKE_BINARY_DIR}/lib/python/nmodl/ DESTINATION nmodl)
```

Comment:
note that with the next PR there will be a shared library and hence we have to change this back to `/lib`.

Reply:
eh ok